### DISCUSSION TITLE
Empty markers_dict using find_diff_features

### DISCUSSION THREAD

**Discussion by MVemon** [aertslab/scenicplus#340] [Q&A]
Hello,

I was running my separate scRNA-seq and scATAC-seq data but encountered a hurdled when inferring candidate enhancer regions. The first couple of steps (impute_accessibility, normalize_scores, and find_highly_variable_features) appear to run fine and do indeed contain regions, but when using find_diff_features, it creates an empty markers_dict list. 

Output of markers_dict:

{'0': Empty DataFrame
 Columns: [Log2FC, Adjusted_pval, Contrast]
 Index: [],
 '1': Empty DataFrame
 Columns: [Log2FC, Adjusted_pval, Contrast]
 Index: [],
 '2': Empty DataFrame
 Columns: [Log2FC, Adjusted_pval, Contrast]
 Index: []}

I wonder if it's due to the low yield of regions in the data?:

CistopicObject from project SRC1 with n_cells × n_regions = 3521 × 288

I tried comparing the cistopic tutorial object and made sure the cell_data columns were in the same order, but still creates an empty markers_dict.

Output of cistopic.cell_data:

![image](https://github.com/aertslab/scenicplus/assets/153677933/7acac221-e8c7-4a50-8a88-da4ebaf26a53)

![image](https://github.com/aertslab/scenicplus/assets/153677933/3911ac04-72c9-4179-8a5d-22fc9664d107)

Any suggestions would be greatly appreciated, thanks!

**Comment by SeppeDeWinter** [comment]
Hi @marck1198 

No sure why this is happening for you. However, I noticed that you only have 288 regions ... this is really not a lot. 
This could also be the cause that you are not finding any DARs.

All the best,

Seppe
